components/engine/src/rocks/util/engine_metrics.rs

```diff
@@ -427,471 +452,340 @@ pub fn flush_engine_ticker_metrics(t: TickerType, value: u64, name: &str) {
-                .set(value.max);
+            engine_histogram_metrics!(
+                STORE_ENGINE_STALL_L0_SLOWDOWN_COUNT_VEC,
+                "stall_l0_slowdown_count",
+                name,
+                value
+            );
         }
         HistType::StallMemtableCompactionCount => {
-            STORE_ENGINE_STALL_MEMTABLE_COMPACTION_COUNT_VEC
-                .with_label_values(&[name, "stall_memtable_compaction_count_median"])
-                .set(value.median);
-            STORE_ENGINE_STALL_MEMTABLE_COMPACTION_COUNT_VEC
-                .with_label_values(&[name, "stall_memtable_compaction_count_percentile95"])
-                .set(value.percentile95);
-            STORE_ENGINE_STALL_MEMTABLE_COMPACTION_COUNT_VEC
-                .with_label_values(&[name, "stall_memtable_compaction_count_percentile99"])
-                .set(value.percentile99);
-            STORE_ENGINE_STALL_MEMTABLE_COMPACTION_COUNT_VEC
-                .with_label_values(&[name, "stall_memtable_compaction_count_average"])
-                .set(value.average);
-            STORE_ENGINE_STALL_MEMTABLE_COMPACTION_COUNT_VEC
-                .with_label_values(&[name, "stall_memtable_compaction_count_standard_deviation"])
-                .set(value.standard_deviation);
-            STORE_ENGINE_STALL_MEMTABLE_COMPACTION_COUNT_VEC
-                .with_label_values(&[name, "stall_memtable_compaction_count_max"])
-                .set(value.max);
+            engine_histogram_metrics!(
+                STORE_ENGINE_STALL_MEMTABLE_COMPACTION_COUNT_VEC,
+                "stall_memtable_compaction_count",
+                name,
+                value
+            );
         }
         HistType::StallL0NumFilesCount => {
-            STORE_ENGINE_STALL_LO_NUM_FILES_COUNT_VEC
-                .with_label_values(&[name, "stall_l0_num_files_count_median"])
-                .set(value.median);
-            STORE_ENGINE_STALL_LO_NUM_FILES_COUNT_VEC
-                .with_label_values(&[name, "stall_l0_num_files_count_percentile95"])
-                .set(value.percentile95);
-            STORE_ENGINE_STALL_LO_NUM_FILES_COUNT_VEC
-                .with_label_values(&[name, "stall_l0_num_files_count_percentile99"])
-                .set(value.percentile99);
-            STORE_ENGINE_STALL_LO_NUM_FILES_COUNT_VEC
-                .with_label_values(&[name, "stall_l0_num_files_count_average"])
-                .set(value.average);
-            STORE_ENGINE_STALL_LO_NUM_FILES_COUNT_VEC
-                .with_label_values(&[name, "stall_l0_num_files_count_standard_deviation"])
-                .set(value.standard_deviation);
-            STORE_ENGINE_STALL_LO_NUM_FILES_COUNT_VEC
-                .with_label_values(&[name, "stall_l0_num_files_count_max"])
-                .set(value.max);
+            engine_histogram_metrics!(
+                STORE_ENGINE_STALL_LO_NUM_FILES_COUNT_VEC,
+                "stall_l0_num_files_count",
```

Comment:
Is this supposed to be `LO` (ell-oh) or `L0` (ell-zero)? 😂

Reply:
😂nice catch